### PR TITLE
rhornung67/doc updates

### PR DESCRIPTION
**Pull request by rhornung67**
# Summary

- This draft PR is a work-in-progress to tidy up the Sidre user documentation. So far I've gone through the Core and Sidre components.  So only take a look at those for now.

I've attempted to clarify the text, fix typos and grammar, update content when I've found it to be out of date, added links, and make formatting consistent. Specifically, I've highlighted class names when the text discusses them as classes. I've left them as regular text when the text discusses them conceptually. Please take a look and let me know what you think. I wanted to get some feedback before tacking more of the documentation. I will probably ignore the inlet and, soon to appear, klee component documentation since those are works in progress.

Here is a link to the sphinx-generated docs so you can see what it looks like https://axom.readthedocs.io/en/rhornung67-doc-updates/

### CODE REVIEW

File: src/axom/sidre/docs/sphinx/parallel_io_concepts.rst [hunk 1]
```diff
@@ -10,68 +10,67 @@ Parallel File I/O
...
-To accomplish parallel I/O, Sidre provides the IOManager class.  This class is
-instantiated with an MPI communicator and provides several overloads of the
-``write()`` and ``read()`` methods.  These methods save a Group in parallel
-to a set of files and read a Group from existing files.
-IOManager optionally uses the SCR library for scalable I/O
-management (such as using burst buffers if available).
+A Sidre ``IOManager`` object is created with an MPI communicator and provides 
+various overloads of ``write()`` and ``read()`` methods.  These methods save 
+a ``Group`` object in parallel to a set of files and read a ``Group`` object
+from existing files. The I/O manager can optionally use the SCR library for
```
Comment (kennyweiss) [inline src/axom/sidre/docs/sphinx/parallel_io_concepts.rst:42]: Should we add a link to SCR?

Perhaps one of: 
* https://computing.llnl.gov/projects/scalable-checkpoint-restart-for-mpi
* https://github.com/LLNL/scr

File: src/axom/sidre/docs/sphinx/parallel_io_concepts.rst [hunk 2]
```diff
@@ -110,48 +109,51 @@ The core functionality of IOManager is contained in the ``write()`` and ``read()
...
+the user-allocated arrays.
+
+Below is a code example for loading external data. We assume that this code
 somehow has knowledge that the root group contains a single external view at
-the location "fields/external_array" describing an array of doubles.  See the
-Group and View documentation for information about how to query the Sidre data
-structures for this type of information when the code does not have a priori
-knowledge.
+the location "fields/external_array" describing an array of doubles. See the
+`Sidre API Documentation <../../../../doxygen/html/sidretop.html>`_ for details
```
Comment (kennyweiss) [inline src/axom/sidre/docs/sphinx/parallel_io_concepts.rst:154]: > See the `Sidre API Documentation <../../../../doxygen/html/sidretop.html>`_ for details

(I commented on a previous link to the doxygen that a link was not properly resolved).

This one *does* link to the expected doxygen page: 
https://axom.readthedocs.io/en/rhornung67-doc-updates/axom/sidre/docs/sphinx/parallel_io_concepts.html#iomanager-class-use

There's probably a subtle difference.

File: src/axom/sidre/docs/sphinx/parallel_io_concepts.rst [hunk 3]
```diff
@@ -178,14 +180,14 @@ knowledge.
...
-bookkeeping information that is used when reading data, but it can also
+The root file is automatically created to provide the ``IOManager`` object 
+with bookkeeping information that is used when reading data, but it can also
 be used to store additional data that may be useful to the calling code or
 is needed to allow other tools to interact with the data in the output files,
-such as for visualization.  For example, Conduit's blueprint index can be
-:ref:`stored in a DataGroup <sidre-conduit>` written to the root file
-to provide metadata about the
-mesh layout and data fields that can be visualized from the output files.
+such as for visualization.  For example, a ``Conduit Mesh Blueprint <https://llnl-conduit.readthedocs.io/en/latest/blueprint.html>`_ index can be
```
Comment (kennyweiss) [inline src/axom/sidre/docs/sphinx/parallel_io_concepts.rst:187]: This link is broken. I think the following will work:

```suggestion
such as for visualization.  For example, a `Conduit Mesh Blueprint <https://llnl-conduit.readthedocs.io/en/latest/blueprint.html>`_ index can be 
```

**Review by kennyweiss — APPROVED** [review]
This looks really nice @rhornung67 

I added a few suggestions and corrections.

File: src/axom/sidre/docs/sphinx/view.rst [hunk 4]
```diff
@@ -9,113 +9,117 @@
...
-          provided for this. The View constructor and destructor are private.
+.. note:: ``View`` objects can only be created and destroyed using ``Group``
+          class methods. The ``View`` class constructor and destructor are 
+          private.
 
-Each View object has a name and is owned by one Group in a Sidre Group 
-hierarchy; its *owning* Group. A View maintains a pointer to the Group that 
-owns it.
+Each view has a name and is owned by one group in a Sidre group hierarchy; i.e.,
+its *owning* Group. A view maintains a pointer to the group that owns it.
```
Comment (agcapps) [inline src/axom/sidre/docs/sphinx/view.rst:24]: Perhaps change "owning Group" to "owning group" to match the capitalization earlier in the sentence.  This capitalization fix may need to propagate throughout the file (such as line 27).

File: src/axom/sidre/docs/sphinx/group.rst [hunk 5]
```diff
@@ -9,53 +9,53 @@
...
-
-.. note:: View copy operations perform **shallow** copies of the View data.
-
-Some methods for creating, destroying, querying, and retrieving Groups and
-Views take a string with *path syntax*, where parent and child Group names
-are joined with the path separator character, '/'.
-Other methods take the name of an
-immediate child of a Group. Methods that require the name of a direct child 
-are marked with 'Child' in their name, such as ``hasChildView()`` and 
+of groups and giews within the moved group is moved as well and can no longer
```
Comment (agcapps) [inline src/axom/sidre/docs/sphinx/group.rst:44]: I'm sure you meant
```suggestion
of groups and views within the moved group is moved as well and can no longer
```

File: src/axom/sidre/docs/sphinx/group.rst [hunk 6]
```diff
@@ -9,53 +9,53 @@
...
-Views take a string with *path syntax*, where parent and child Group names
-are joined with the path separator character, '/'.
-Other methods take the name of an
-immediate child of a Group. Methods that require the name of a direct child 
-are marked with 'Child' in their name, such as ``hasChildView()`` and 
+of groups and giews within the moved group is moved as well and can no longer 
+be accessed via the original parent group. When a group is copied to another 
+group, a copy of the entire group subtree rooted at the copied group is added
+to the group to which it is copied. A **shallow** copy is performed for the
+data in each view. SPecifically, a new view objects is created in the
```
Comment (agcapps) [inline src/axom/sidre/docs/sphinx/group.rst:48]: double capitalization
```suggestion
data in each view. Specifically, a new view object is created in the
```

**Review by agcapps — APPROVED** [review]
Whew!  Thank you, @rhornung67 .  I flagged a few typos but overall looks good.

File: src/axom/core/docs/sphinx/core_containers.rst [hunk 7]
```diff
@@ -8,50 +8,54 @@ Core containers
...
+``std::vector`` to GPUs.
 
-Array is a multidimensional contiguous container template.  In the 1-dimensional case,
-this class will behave like std::vector.  In higher dimensions some vector-like
-functionality will be unavailable, e.g., ``push_back``, and multidimensional-specific
-operations will mirror the ``ndarray`` provided by ``numpy`` when possible.
-In the future, it will be possible to take "views" of the underlying array data that
-allow for flexible reinterpretation via different striding.
+``Array`` is a multidimensional contiguous container template. In the 
+1-dimensional case, this class behave similar to ``std::vector``. In higher
```
[the remark this answers is not in the document]
Comment (rhornung67) [reply src/axom/core/docs/sphinx/core_containers.rst:21]: fixed.

File: src/axom/core/docs/sphinx/core_containers.rst [hunk 8]
```diff
@@ -8,50 +8,54 @@ Core containers
...
+``push_back``, and multidimensional-specific operations will mirror the 
+``ndarray`` provided by ``numpy`` when possible. In the future, it will be 
+possible to take "views" of the underlying array data that allow for 
+flexible reinterpretation via different striding.
 
-MCArray (or Multi-Component Array) is a container template for a contiguous array of tuples.
-In this sense it can be thought of as a two-dimensional array.  MCArray will be removed
-imminently as Array provides/will provide this multidimensional functionality.
+``MCArray`` (or Multi-Component Array) is a container template for a contiguous
+array of tuples. In this sense, it can be thought of as a two-dimensional array.``MCArray`` will be removed in the future as ``Array`` will provide all the
```
[the remark this answers is not in the document]
Comment (rhornung67) [reply src/axom/core/docs/sphinx/core_containers.rst:29]: fixed.

File: src/axom/core/docs/sphinx/core_containers.rst [hunk 9]
```diff
@@ -8,50 +8,54 @@ Core containers
...
+array of tuples. In this sense, it can be thought of as a two-dimensional array.``MCArray`` will be removed in the future as ``Array`` will provide all the 
+this multidimensional functionality of ``MCArray``..
 
-Here's an example showing how to use MCArray instead of `std::vector`.
+Here's an example showing how to use ``MCArray`` instead of ``std::vector``.
 
 .. literalinclude:: ../../examples/core_containers.cpp
    :start-after: _arraybasic_start
    :end-before: _arraybasic_end
    :language: C++
```
[the remark this answers is not in the document]
Comment (rhornung67) [reply src/axom/core/docs/sphinx/core_containers.rst:38]: I added a bit more output and included the output of the array examples in the docs.

File: src/axom/core/docs/sphinx/core_containers.rst [hunk 10]
```diff
@@ -8,50 +8,54 @@ Core containers
...
 
-MCArray (or Multi-Component Array) is a container template for a contiguous array of tuples.
-In this sense it can be thought of as a two-dimensional array.  MCArray will be removed
-imminently as Array provides/will provide this multidimensional functionality.
+``MCArray`` (or Multi-Component Array) is a container template for a contiguous
+array of tuples. In this sense, it can be thought of as a two-dimensional array.``MCArray`` will be removed in the future as ``Array`` will provide all the 
+this multidimensional functionality of ``MCArray``..
 
-Here's an example showing how to use MCArray instead of `std::vector`.
+Here's an example showing how to use ``MCArray`` instead of ``std::vector``.
```
[the remark this answers is not in the document]
Comment (rhornung67) [reply src/axom/core/docs/sphinx/core_containers.rst:32]: I left the examples in the docs as is. If I change it now, then the docs will need to be changed again when axom::Array is removed and it's functionality is subsumed by the expanded MCArray class.

File: src/axom/sidre/docs/sphinx/mfem_sidre_datacollection.rst [hunk 11]
```diff
@@ -10,122 +10,142 @@ Using Sidre with MFEM
...
+to the Mesh Blueprint, a hierarchical schema for describing mesh data.
 
-See the :ref:`Conduit page <sidre-conduit>` for more information on the Mesh Blueprint.
+See the :ref:`Conduit page <sidre-conduit>` for more information on the Mesh 
+Blueprint.
 
-In this document, we first discuss `Getting Started`_ and how MFEM objects can be associated with the ``DataCollection``.
-We then explain the process for and options available when `Saving Data to a File`_.
-The workflow for reading saved data back in is discussed in `Restarting a Simulation`_.
+In this document, we first discuss `Getting Started`_ and how MFEM objects can
```
[the remark this answers is not in the document]
Comment (rhornung67) [reply src/axom/sidre/docs/sphinx/mfem_sidre_datacollection.rst:28]: fixed

File: src/axom/sidre/docs/sphinx/mfem_sidre_datacollection.rst [hunk 12]
```diff
@@ -10,122 +10,142 @@ Using Sidre with MFEM
...
 
-In this document, we first discuss `Getting Started`_ and how MFEM objects can be associated with the ``DataCollection``.
-We then explain the process for and options available when `Saving Data to a File`_.
-The workflow for reading saved data back in is discussed in `Restarting a Simulation`_.
+In this document, we first discuss `Getting Started`_ and how MFEM objects can 
+be associated with the ``MFEMSidreDataCollection``. We then explain the process
+and options available when `Saving Data to a File`_. The workflow for reading 
+saved data back in is discussed in `Restarting a Simulation`_.
 
 Getting Started
```
[the remark this answers is not in the document]
Comment (rhornung67) [reply src/axom/sidre/docs/sphinx/mfem_sidre_datacollection.rst:33]: fixed

File: src/axom/sidre/docs/sphinx/parallel_io_concepts.rst (hunk 1 above)
Comment (rhornung67) [reply src/axom/sidre/docs/sphinx/parallel_io_concepts.rst:42]: done.

File: src/axom/sidre/docs/sphinx/parallel_io_concepts.rst (hunk 3 above)
Comment (rhornung67) [reply src/axom/sidre/docs/sphinx/parallel_io_concepts.rst]: Fixed. extra '`' at the beginning.

File: src/axom/sidre/docs/sphinx/parallel_io_concepts.rst (hunk 2 above)
Comment (rhornung67) [reply src/axom/sidre/docs/sphinx/parallel_io_concepts.rst:155]: see my comment on the broken link fix.

File: src/axom/core/docs/sphinx/core_containers.rst [hunk 13]
```diff
@@ -8,50 +8,54 @@ Core containers
...
    :start-after: _extbuffer_start
    :end-before: _extbuffer_end
    :language: C++
 
-Similar to `std::vector`, when using an internal array the Array and MCArray classes can
-reserve memory in anticipation of future growth as well as shrink to just the
-memory currently in use.
+Similar to ``std::vector``, when using an internal array the ``Array`` and 
+``MCArray`` classes can reserve memory in anticipation of future growth as 
+well as shrink to the memory in use.
```
[the remark this answers is not in the document]
Comment (rhornung67) [reply src/axom/core/docs/sphinx/core_containers.rst]: I added a link to the doxygen source dox for completeness and easier maintenance.

File: src/axom/core/docs/sphinx/core_utilities.rst [hunk 14]
```diff
@@ -7,10 +7,11 @@
 Core utilities
 ******************************************************
 
-The `axom::utilities` namespace contains basic useful functions.  Often these
-have started out in another, higher-level library and proven so useful they're
-"promoted" to Axom Core.  In some cases, `axom::utilities` brings functionality
-from recent standards of C++ to platforms restricted to older compilers.
+The ``axom::utilities`` namespace contains basic commonly-used functions.
+Often these started out in another, Axom library and were "promoted" to Axom
```
[the remark this answers is not in the document]
Comment (rhornung67) [reply src/axom/core/docs/sphinx/core_utilities.rst:11]: fixed.

File: src/axom/core/docs/sphinx/core_utilities.rst [hunk 15]
```diff
@@ -36,8 +37,8 @@ filesystem example snippet.
    :language: C++
 
 There are several other utility functions.  Some are numerical functions such as
```
[the remark this answers is not in the document]
Comment (rhornung67) [reply src/axom/core/docs/sphinx/core_utilities.rst:39]: I added a note on why we provide these and also a link to the doxygen API docs.

File: src/axom/core/docs/sphinx/index.rst [hunk 16]
```diff
@@ -7,19 +7,18 @@ Axom Core User Guide
 ====================
 
 The Axom Core library provides fundamental data structures and operations used
```
[the remark this answers is not in the document]
Comment (rhornung67) [reply src/axom/core/docs/sphinx/index.rst:9]: fixed here and did a quick pass through and changed from library to component where needed.

File: src/axom/core/docs/sphinx/index.rst [hunk 17]
```diff
@@ -7,19 +7,18 @@ Axom Core User Guide
...
-namespace.  Axom Core also contains the `axom::Array` and `axom::StackArray`
+the ``axom::numerics`` namespace.  It contains various utilities including a 
+timer class, lexical comparator class, ``processAbort()`` function, and numeric,
+filesystem, and string manipulation utilities in the ``axom::utilities``
+namespace.  Axom Core also contains the ``axom::Array`` and ``axom::StackArray``
 container classes.
 
-The Core library is used by the rest of Axom, so Core does not use macros from
-Axom Slic for output.
+The Core library does not depend on any other Axom components/libraries. In
```
[the remark this answers is not in the document]
Comment (rhornung67) [reply src/axom/core/docs/sphinx/index.rst:20]: made it a note.

File: src/axom/sidre/docs/sphinx/buffer.rst [hunk 18]
```diff
@@ -9,43 +9,43 @@
...
-   total number of bytes, number of bytes per element, etc.
- * Retrieve the number of Views the Buffer is attached to.
- * Copy Buffer description and its data to/from a conduit::Node.
+.. note:: * ``Buffer`` objects can only be created and destroyed using 
+            ``DataStore`` class methods. Buffer constructor and destructor 
+            methods are private (see :ref:`datastore-label`).
+          * Each ``Buffer`` object has a unique integer identifier generated 
+            when the buffer is created. If you want to retrieve a buffer or
+            interact with it directly, you must keep a pointer to it or note 
+            its id so that you can retrieve it from a dataStore when needed.
```
[the remark this answers is not in the document]
Comment (rhornung67) [reply src/axom/sidre/docs/sphinx/buffer.rst:28]: Done. Made it "datastore" consistently throughout.

File: src/axom/sidre/docs/sphinx/core_concepts.rst [hunk 19]
```diff
@@ -7,15 +7,16 @@
...
-Attribute. In combination, these classes implement a data store with a tree 
-structure to organize data in a hierarchy:
-
-* DataStore is the main interface to access a data hierarchy.
-* Buffer describes and holds data in memory.
-* Group defines parent-child relationships in a hierarchical tree data structure and provides access to file I/O operations.
-* View provides a virtual description of data and access to it.
-* Attribute allows a program to attach metadata to View objects for processing data selectively. 
+As noted earlier and described in the previous example, Sidre provides five 
+main classes: ``Datastore``, ``Buffer``, ``Group``, ``View``, and ``Attribute``.In combination, these classes implement a data store with a tree structure
```
[the remark this answers is not in the document]
Comment (rhornung67) [reply src/axom/sidre/docs/sphinx/core_concepts.rst:11]: fixed

File: src/axom/sidre/docs/sphinx/file_io.rst [hunk 20]
```diff
@@ -7,51 +7,55 @@
...
+but in an external data allocation.
+
+Overloads of the ``save()`` and ``load()`` methods that take HDF5 handles and 
+the ``loadExternalData()`` method are used to implement parallel I/O through 
+the Sidre ``IOManager`` class.
 
-The overloads of ``save()`` and ``load()`` that take HDF5 handles and the
-``loadExternalData()`` method are public APIs that are used to implement parallel
-I/O through the IOManager class.
+Please see `Sidre API Documentation <../../../../doxygen/html/sidretop.html>`
```
[the remark this answers is not in the document]
Comment (rhornung67) [reply src/axom/sidre/docs/sphinx/file_io.rst:58]: fixed. missing '_' at the end.

File: src/axom/sidre/docs/sphinx/group.rst [hunk 21]
```diff
@@ -66,154 +66,155 @@ is equivalent to::
...
    other descendant).
- * Destroy all child groups in a Group.
+ * Destroy all child groups in a group.
 
-.. note:: When a Group is destroyed, all Groups and Views in the subtree 
-          rooted at the destroyed Group are also destroyed. However, the 
-          data associated with those Views will remain intact.
+.. note:: When a ``Group`` object is destroyed, all groups and views in the 
+          subtree rooted at the destroyed group are also destroyed. However, 
+          the data associated with the views will remain intact.
```
[the remark this answers is not in the document]
Comment (rhornung67) [reply src/axom/sidre/docs/sphinx/group.rst:104]: I added an issue and tagged it to the milestone for the next release.

File: src/axom/sidre/docs/sphinx/group.rst [hunk 22]
```diff
@@ -66,154 +66,155 @@ is equivalent to::
...
+ * Save and load group subtrees, including data in associated views, to and
    from files. A variety of methods are provided to support different I/O
    operations, different I/O protocols, etc.
 
-I/O methods on the Group class use `Conduit <https://github.com/LLNL/conduit>`_
-to :ref:`write the data (sub)tree <sidre-serial-io>` rooted in a Group to a 
+I/O methods on the group class use `Conduit <https://github.com/LLNL/conduit>`_
+to :ref:`write the data (sub)tree <sidre-serial-io>` rooted in a group to a 
 file, `HDF5 <https://www.hdfgroup.org/HDF5/>`_ handle, or other
 Conduit protocol, or to an in-memory Conduit data structure. An application may
```
[the remark this answers is not in the document]
Comment (rhornung67) [reply src/axom/sidre/docs/sphinx/group.rst:216]: done.

File: src/axom/core/docs/sphinx/core_containers.rst [hunk 23]
```diff
@@ -8,50 +8,54 @@ Core containers
...
-Array is a multidimensional contiguous container template.  In the 1-dimensional case,
-this class will behave like std::vector.  In higher dimensions some vector-like
-functionality will be unavailable, e.g., ``push_back``, and multidimensional-specific
-operations will mirror the ``ndarray`` provided by ``numpy`` when possible.
-In the future, it will be possible to take "views" of the underlying array data that
-allow for flexible reinterpretation via different striding.
+``Array`` is a multidimensional contiguous container template. In the 
+1-dimensional case, this class behave similar to ``std::vector``. In higher 
+dimensions, some vector-like functionality will be unavailable, e.g., 
+``push_back``, and multidimensional-specific operations will mirror the
```
[the remark this answers is not in the document]
Comment (rhornung67) [reply src/axom/core/docs/sphinx/core_containers.rst:23]: I, fixed, it.

File: src/axom/core/docs/sphinx/core_containers.rst [hunk 24]
```diff
@@ -8,50 +8,54 @@ Core containers
...
+``ndarray`` provided by ``numpy`` when possible. In the future, it will be 
+possible to take "views" of the underlying array data that allow for 
+flexible reinterpretation via different striding.
 
-MCArray (or Multi-Component Array) is a container template for a contiguous array of tuples.
-In this sense it can be thought of as a two-dimensional array.  MCArray will be removed
-imminently as Array provides/will provide this multidimensional functionality.
+``MCArray`` (or Multi-Component Array) is a container template for a contiguous
+array of tuples. In this sense, it can be thought of as a two-dimensional array.``MCArray`` will be removed in the future as ``Array`` will provide all the 
+this multidimensional functionality of ``MCArray``..
```
[the remark this answers is not in the document]
Comment (rhornung67) [reply src/axom/core/docs/sphinx/core_containers.rst:31]: fixed.

File: src/axom/sidre/docs/sphinx/attribute.rst [hunk 25]
```diff
@@ -9,30 +9,32 @@
...
-            methods. The Attribute constructor and destructor are private 
-            (see :ref:`datastore-label`).
-          * Each Attribute has a unique name and integer identifier. Either can
-            be used to retrieve it from the DataStore.
+.. note:: * ``Attribute`` objects can only be created and destroyed using 
+            ``DataStore`` class methods. The ``Attribute`` class constructor 
+            and destructor are private (see :ref:`datastore-label`).
+          * Each ``Attribute`` object has a unique name and integer identifier.
+            Either can be used to retrieve an ``Attribute`` object from a 
+            dataStore.
```
[the remark this answers is not in the document]
Comment (rhornung67) [reply src/axom/sidre/docs/sphinx/attribute.rst:26]: done.

File: src/axom/sidre/docs/sphinx/datastore.rst [hunk 26]
```diff
@@ -9,35 +9,37 @@
...
+a collection of ``Attribute`` objects. Generally, the first thing a Sidre user 
+does is create the datastore object; this operation also creates the root group.
+Apart from providing access to the root group, a dataStore provides methods to 
+retrieve and interact with buffers and attributes.
+
+.. note:: ``Buffer`` and ``Attribute`` objects can only be created and 
+          destroyed using ``DataStore`` methods noted below. The ``Buffer`` and
+          ``Attribute`` class constructors and destructors are private.
+
+The ``Datastore`` class provides the following methods to manage and interact
```
[the remark this answers is not in the document]
Comment (rhornung67) [reply src/axom/sidre/docs/sphinx/datastore.rst:24]: fixed throughout the docs.

File: src/axom/sidre/docs/sphinx/view.rst [hunk 27]
```diff
@@ -9,113 +9,117 @@
 View
 ==========
 
-A Sidre View describes data and provides access to it. A View can describe
-a (portion) of a data allocation in **any way that is compatible** with the
-allocation. Specifically, the allocation must contain enough bytes to support
-the description. In particular, the data type of a View description need
-not match the types associated with the allocation employed by other Views
-into that data.
+A Sidre ``View`` objects describes data and provides access to it. A view can
```
[the remark this answers is not in the document]
Comment (rhornung67) [reply src/axom/sidre/docs/sphinx/view.rst:12]: fixed.

File: src/axom/sidre/docs/sphinx/view.rst [hunk 28]
```diff
@@ -9,113 +9,117 @@
...
 ==========
 
-A Sidre View describes data and provides access to it. A View can describe
-a (portion) of a data allocation in **any way that is compatible** with the
-allocation. Specifically, the allocation must contain enough bytes to support
-the description. In particular, the data type of a View description need
-not match the types associated with the allocation employed by other Views
-into that data.
+A Sidre ``View`` objects describes data and provides access to it. A view can 
+describe a (portion) of a data allocation in **any way that is compatible**
```
[the remark this answers is not in the document]
Comment (rhornung67) [reply src/axom/sidre/docs/sphinx/view.rst:13]: fixed.

File: src/axom/sidre/docs/sphinx/view.rst (hunk 4 above)
Comment (rhornung67) [reply src/axom/sidre/docs/sphinx/view.rst:24]: Good catch. fixed.

File: src/axom/sidre/docs/sphinx/group.rst (hunk 5 above)
Comment (rhornung67) [reply src/axom/sidre/docs/sphinx/group.rst:44]: fixed. although 'view' is an overloaded term....

File: src/axom/sidre/docs/sphinx/group.rst (hunk 6 above)
Comment (rhornung67) [reply src/axom/sidre/docs/sphinx/group.rst:48]: FIxed.

Comment (agcapps) [reply src/axom/sidre/docs/sphinx/group.rst]: THanks!

Comment (rhornung67) [reply src/axom/sidre/docs/sphinx/group.rst]: YOu're WElcome

File: src/axom/core/doxygen_mainpage.md [hunk 29]
```diff
@@ -6,8 +6,15 @@ These hide the difference between versions of C++ and between supported computin
...
   - A [Timer](@ref axom::utilities::Timer) class to measure elapsed time
+  - [File system operations](@ref axom::utilities::filesystem)
+  - [String manipulation](@ref axom::utilities::string) 
+  - Routines for getting information about the machine and user running the code (axom::utilities::getHostName() and axom::utilities::getUserName() methods) and gracefully exiting execution (processAbout() method)
+  - Routines for getting information about the Axom version (axom::getVersion() method) and how Axom was configured and built (axom::about() method)
   - Templated arithmetic and logical comparison functions
-  - A processAbort() function
+- [Containers]
+  - [Basic linear array](@ref axom::Array)
+  - [Mutli-component array](@ref axom::MCArray)
```
Comment (white238) [inline src/axom/core/doxygen_mainpage.md:16]: "Mutli" to "Multi"

Comment (rhornung67) [reply src/axom/core/doxygen_mainpage.md]: Fixed.

**Comment by rhornung67** [comment]
I've addressed all reviewer comments, so this should be good to merge. Does anyone else want to review?

**Pull request merged**